src/test/java/core/basesyntax/file/writer/FileWriterImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.file.writer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileWriterImplTest {
+    private FileWriter fileWriter;
+
+    @BeforeEach
+    void setUp() {
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    void write_validFile_ok() throws IOException {
+        Path tempFile = Files.createTempFile("output", ".csv");
+
+        String data = "fruit,quantity\nbanana,20";
```

Comment:
why it`s valid?

Reply:
Not fixed
Change data content according to the task requirements